variables.tf

```diff
@@ -29,7 +44,8 @@ variable "private_subnets" {
       "nat" : true,
       "vsphere_service_type" : "management",
       "routable" : true,
-      "cidr" : "172.16.0.0/24"
+      "cidr" : "172.16.0.0/24",
+      "reserved_ip_count" = 100
```

Comment:
Let's keep the syntax the same here and use `:` instead of `=`.

Reply:
Agreed.